documentation/meta/release_and_deployment/index.md

```diff
@@ -0,0 +1,411 @@
+
+Roll back and redeployment of the API and frontend follows an identical process,
+based on the approach of forcing a deployment of the app to a specific version.
+Regardless of whether that version is the version currently running, a past
+version, or a completely new version, the same deployment workflows are used in
+all cases. The same process works to retry a failed deployment.
+
+To force a deployment of the API or frontend (of either environment) to a
+specific version, manually trigger the deployment workflow for the app and
+environment with the docker image tag that should be deployed.
+
+To find historically deployed production image tags, refer to the title of the
+workflow runs for the app (linked in the table below). The `rel-*` image tags in
+the titles of the production deployment workflow runs is the image tag deployed
+for that run.
+
+To find the currently deployed tag of any environment, visit the `/version`
+endpoint of the API or frontend environment you wish to redeploy:
+
+| App version endpoint                                                 | Deployment workflow                                                                                                           |
+| -------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| [Frontend staging `/version`](https://staging.openverse.org/version) | [Deployment workflow](https://github.com/WordPress/openverse-infrastructure/actions/workflows/deploy-staging-frontend.yml)    |
+| [Frontend production `/version`](https://openverse.org/version)      | [Deployment workflow](https://github.com/WordPress/openverse-infrastructure/actions/workflows/deploy-production-frontend.yml) |
+| [API staging `/version`](https://api-staging.openverse.org/version)  | [Deployment workflow](https://github.com/WordPress/openverse-infrastructure/actions/workflows/deploy-staging-api.yml)         |
+| [API production `/version`](https://api.openverse.org/version)       | [Deployment workflow](https://github.com/WordPress/openverse-infrastructure/actions/workflows/deploy-production-api.yml)      |
+
+Click the "Run workflow" button to open the manual workflow trigger dialogue.
+
+```{figure} ./manually-trigger-workflow.png
+:alt: Deployment workflow manual trigger dialogue
+:height: 400
+```
+
+Enter the tag found from the `/version` endpoint as the "Image tag to deploy" in
+the manual workflow run dialogue, add a reason for the manual deployment as the
+"name of the workflow run", and then click the dialogue's "Run workflow" button.
+
+```{admonition} Production tag details
+:class: warning
+
+The production release and deployment process uses the <code>rel-*</code> pattern image tags when dispatching
+the deployment workflow for each app. However, the <code>rel-*</code> image tags are merely aliases for the
```

Comment:
Note that the `*` characters here are causing some odd formatting in the rendered documentation:
![image](https://github.com/user-attachments/assets/24160732-727a-4a17-9d24-9ab8c33bc7cf)

Reply:
Oops, good catch. They can be escaped with `\` to fix this.